### ISSUE TITLE
stderr log file

### ISSUE THREAD

**Issue by nbari** [immortal/immortal#22]
Be available to specify a file only for writing logs related only to `stderr`, probably something like this:

```yaml
    log:
        file: /var/log/app/access.log
        age: 86400
        num: 3
        size: 1
    stderr_log:
        file: /var/log/app/error.log
        age: 86400
        num: 3
        size: 1
```

The complexity of this is the creation of another pair of pipes only for the `stderr` and another logger that both file/err should be rotated on the defined period

**Comment by nbari** [comment]
Create 3 log options:

```yaml
log:
  file: /var/log/app/access.log
  age: 86400
  num: 3
  size: 1
stdout_log:
  file: /var/log/app/error.log
  age: 86400
  num: 3
  size: 1
stderr_log:
  file: /var/log/app/error.log
  age: 86400
  num: 3
  size: 1
```

If `log`,  stdout & stderr will be combined. (default and used also for a custom `logger`)

if `stderr_log`, the log will contain only standard error

if `stdout_log`, the log will contain only standard output

Depending on what is required, any or all of  3 options can be used.

**Comment by nbari** [comment]
implemented on 0.16.0
